projects/data-grid/src/lib/constants/translation-keys.ts

```diff
@@ -9,8 +9,7 @@ export const LOCALE_TEXT_KEYS = {
     },
     pagination: {
       itemsPerPage: 'grid.pagination.itemsPerPage',
-      of: 'grid.pagination.of',
-      items: 'grid.pagination.items'
```

Comment:
The key should be removed also from:
 - translation service
 - default-locale-tests
 - locale-texts